picasso/src/main/java/com/squareup/picasso3/Request.java

```diff
@@ -640,6 +647,12 @@ public Builder networkPolicy(@NonNull NetworkPolicy policy,
       return this;
     }
 
+    @NonNull
+    public Builder addHeaders(@Nullable Headers headers) {
```

Comment:
i think we'll eventually want to make it easy to add and remove headers.
like exposing `public Headers.Builder Request.Builder.headers`.

and then the Headers at `Request.headers` is never null.

Reply:
@NightlyNexus should i change this method so that it exposes Headers.Builder ? 
Like,  public Headers.Builder addHeaders(..) ?

Reply:
@NightlyNexus @JakeWharton Could you please give me clarification on this?

Reply:
No. The method should take two `String`s. Most people don't care how the headers are actually represented.

Reply:
@JakeWharton @NightlyNexus updated the method to collect two Strings.